packages/pluggableWidgets/maps-native/src/Maps.editorConfig.ts

```diff
@@ -32,14 +103,61 @@ export function check(values: MapsPreviewProps): Problem[] {
         });
     }
 
-    return errors;
-}
+    values.markers.forEach((marker, index) => {
+        if (marker.locationType === "address") {
+            if (!marker.address) {
+                errors.push({
+                    property: `markers/${index + 1}/address`,
+                    message: "A static marker requires an address"
+                });
+            }
+        } else {
+            if (!marker.latitude) {
+                errors.push({
+                    property: `markers/${index + 1}/latitude`,
+                    message: "A static marker requires latitude"
+                });
+            }
+            if (!marker.longitude) {
+                errors.push({
+                    property: `markers/${index + 1}/longitude`,
+                    message: "A static marker requires longitude"
+                });
+            }
+        }
+    });
 
-export function getPreview(_: MapsPreviewProps): StructurePreviewProps {
-    return {
-        type: "Image",
-        document: decodeURIComponent(StructurePreviewMapsSVG.replace("data:image/svg+xml,", "")),
-        width: 375,
-        height: 375
-    };
+    values.dynamicMarkers.forEach((marker, index) => {
+        // @ts-ignore
+        if (marker.markersDS.type === "null") {
```

Comment:
If I am not wrong page editor fixed this issue and now it will be either {} or null in 9.2/9.3